### DISCUSSION TITLE
How do i add my apps on this tool

### DISCUSSION THREAD

**Discussion by SHR1SHAK** [ChrisTitusTech/winutil#2239] [Q&A]
I want to add a cool FOSS tool. Can I just add it to the Application.json and be done ?

**Comment by og-mrk** [comment]
@SHR1SHAK Yes you can.
Simply edit the `applications.json` file under the config folder, (Re)-compile WinUtil by running the `Compile.ps1` script, and try running `winutil.ps1` as admin to see your newly added app.
